tests/sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__scalar_bucket_counts_v1/test_minimal/data.py

```diff
@@ -0,0 +1,164 @@
+"""Testing data for query."""
+from pathlib import Path
+from uuid import uuid4
+
+import yaml
+
+ROOT = Path(__file__).parent
+
+SUBMISSION_DATE = "2020-10-01"
+APP_BUILD_ID = "2020100100"
+
+# Other tests: non * fields
+CLIENTS_SCALAR_AGGREGATES = [
+    {
+        "client_id": str(uuid4()),
+        "ping_type": "*",
+        "os": "*",
+        "app_version": 84,
+        "app_build_id": APP_BUILD_ID,
+        "channel": "*",
+        "scalar_aggregates": [
+            {
+                "metric": "places_manager_write_query_count",
+                "metric_type": "counter",
+                "key": "",
+                "agg_type": "count",
+                "value": 4.0,
+            },
+        ],
+    },
+    {
+        "client_id": str(uuid4()),
+        "ping_type": "*",
+        "os": "*",
+        "app_version": 84,
+        "app_build_id": APP_BUILD_ID,
+        "channel": "*",
+        "scalar_aggregates": [
+            {
+                "metric": "places_manager_write_query_count",
+                "metric_type": "counter",
+                "key": "",
+                "agg_type": "count",
+                "value": 8.0,
+            },
+        ],
+    },
+]
+
+# TODO: why are the range_min and range_max set at these values?
```

Comment:
The behavior of range_min and range_max is not intuitive, there might be something weird going on here.

Reply:
Can clients_scalar_aggregates have `*` values for ping_type and os?  Could this be caused by the cross join with the static combos?  Maybe not because app_build_id is defined.  If I understand correctly, min/max should be 2/3 right?

Reply:
Even with the cross join, I'm not sure how the range max (in log space) could be set to 5 when the max value is 8.

Reply:
I have a feeling it might be related to the cross join and the input data having `"ping_type": "*", "os": "*",` but I don't really know.  If you can't figure out then I don't want to block merging this.  It would be nice to verify in case there's actually an issue in the etl though.

Reply:
It's tangentially related to the cross join. I'm going to follow up with this in a separate PR because this one is already very large for a minimal PR.